lib/rubocop/cop/gusto/explicit_time_units.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Gusto
+      class ExplicitTimeUnits < Base
+        MSG = "Avoid adding/subtracting integers or floats directly to Date/Time/DateTime. " \
+          "Use explicit time methods instead (e.g., `.days`, `.hours`)."
+
+        RESTRICT_ON_SEND = %i(+ - << >>).freeze
```

Comment:
I feel like bit-shifting doesn't really come up in specs. Do we have any examples of this? 

`+ -` should be sufficient if we don't have any existing examples